Project Lifecycle.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Project Lifecycle
+
+## Project Definition
+
+The Node.js Foundation hosts several "Top Level Projects." These projects are autonomous from each other and governed by their own TC (Technical Committee) and chartered by the Node.js Foundation TSC.
+
+Projects are free to create "Working Groups" which are autonomous groups collaborating to fulfill a set of responsibilities. Working Groups are eventually chartered by the TC. The TSC also charters its own Working Groups.
+
+```
+ TSC
+  |
+  |-- Project A TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Project B TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Working Group A (Chartered by TSC)
+  |-- Working Group B (Chartered by TSC)
+```
+
+Both TLPs and TSC WGs may elect a representative to the TSC. TLPs and WGs with *incubation* status are not granted voting privileges on the TSC.
```

Comment:
I'm not sure how I feel about this. Maybe this should be like, variable somehow? "up to two"?